instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+function PostContainer(props) {
+  console.log(props);
+  return (
+    <>
+      {props.data.map(data => (
```

Comment:
The props object has a property called "insta" that you passed down from the App component. *insta* is the array of objects we want to map.
`props.insta.map(data => {`